build/Makefile

```diff
@@ -116,6 +116,9 @@ SYSTEST_CONFIG := $(BASE_DIR)/test/config/systest.conf
 .PHONY: tests test omstest unittest systemtest systemtestrb systemtestsh
 
 all : $(SCX_INTERMEDIATE_DIR) $(DSC_TARGET_DIR) $(RUBY_TESTING_DIR) $(RUBY_DEST_DIR) $(IN_PLUGINS_LIB) kit
+        # After a successful make, undo the Ruby patches for sequential builds from old branches
+	@echo "Cleaning patched Ruby files..."
+	cd $(BASE_DIR)/source/ext/ruby; git checkout -- ext/openssl/ossl_ssl.c ext/openssl/extconf.rb
```

Comment:
This one is to clean up the workaround for ssl issue , correct? if this step is specific to a ruby version, it would be good to have a comment about it, it might no longer be needed if base os is moved to cent6 or ruby is upgraded.